Proiect-VS/assets/css/evaluations.less

```diff
@@ -0,0 +1,91 @@
+body {
+    position: relative;
+    font-family: Arial;
+    background-color: #ffffff; 
+    margin: auto;
+}
+
+header {
+    height: 70px;
+    border-bottom: 1px solid grey;
+}
+
+footer {
+    height: 60px; 
+    position: absolute;
+    left: 0;
+    right: 0;
+    line-height: 60px;
+    background-color: #bcbcaf;
+    color: #686865;
+    text-align: right;
+    padding-right: 50px;
+
+}
+
+a {
+    vertical-align: middle;
+    line-height: 70px;
+    margin-left: 100px;
```

Comment:
these are not generic enought styles to be applied to all a elements( eg margin-left:100px etc). Please use a class to apply these margings and keep in "a" only the truly generic properties.